### PR TITLE
[GF] Fix Sonar warnings

### PR DESCRIPTION
**Pull request by tommydeboer**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

**Comment by dennishendriksen** [comment]
```
Failed tests:
 OntologyTagServiceTest>AbstractTestNGSpringContextTests.run:175->testAddAttributeTag:195 » NullPointer
 OntologyTagServiceTest>AbstractTestNGSpringContextTests.run:175->testRemoveAllTagsFromEntity:257 » UnknownAttribute
 OntologyTagServiceTest>AbstractTestNGSpringContextTests.run:175->testRemoveAttributeTag:220 » NullPointer
 OntologyTagServiceTest>AbstractTestNGSpringContextTests.run:175->testTagAttributesInEntity:288 » NullPointer
 OntologyTagServiceTest>AbstractTestNGSpringContextTests.run:175->testgetTagsForAttribute:141 expected [{instanceOf=[OntologyTerm{IRI=http://edamontology.org/data_0987, label=Chromosome name, description=Name of a chromosome., synonyms=[Name of a chromosome.]}, OntologyTerm{IRI=http://edamontology.org/data_0919, label=Gene annotation (chromosome), description=This includes basic information. e.g. chromosome number..., synonyms=[This includes basic information. e.g. chromosome number...]}]}] but found [{}]
```

File: molgenis-semantic-mapper/src/test/java/org/molgenis/data/mapper/algorithmgenerator/rules/impl/InternalAbstractCategoryRuleTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.molgenis.data.mapper.algorithmgenerator.bean.Category;
+import org.testng.annotations.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class InternalAbstractCategoryRuleTest
+{
+	@Test
+	public void testCreateNumericQualityIndicatorDoesNotThrowNPE()
```
Comment (tommydeboer) [inline molgenis-semantic-mapper/src/test/java/org/molgenis/data/mapper/algorithmgenerator/rules/impl/InternalAbstractCategoryRuleTest.java]: https://sonarcloud.io/project/issues?id=org.molgenis%3Amolgenis&open=AVzJsl-DsY0y2VuVC9aN&resolutions=FALSE-POSITIVE&types=BUG

**Comment by dennishendriksen** [comment]
```
Failed tests: 
  OntologyTagServiceTest>AbstractTestNGSpringContextTests.run:175->testRemoveAttributeTag:214 expected [[sys_md_Tag{id=1234&objectIRI=http://edamontology.org/data_0987&label=Chromosome name&relationIRI=http://molgenis.org/biobankconnect/instanceOf&relationLabel=instanceOf&codeSystem=http://edamontology.org}, sys_md_Tag{id=4321&objectIRI=http://edamontology.org/data_0919&label=Gene annotation (chromosome)&relationIRI=http://molgenis.org/biobankconnect/instanceOf&relationLabel=instanceOf&codeSystem=http://edamontology.org}]] but found [[sys_md_Tag{id=1234&objectIRI=http://edamontology.org/data_0987&label=Chromosome name&relationIRI=http://molgenis.org/biobankconnect/instanceOf&relationLabel=instanceOf&codeSystem=http://edamontology.org}]]
```

File: molgenis-data-import/src/main/java/org/molgenis/data/importer/ImportRunService.java [hunk 2]
```diff
@@ -120,20 +120,25 @@ private String createMailTitle(ImportRun importRun)
...
-			if (importRun != null)
+			try
 			{
 				importRun.setStatus(ImportStatus.FAILED.toString());
 				importRun.setEndDate(now());
 				importRun.setMessage(message);
 				dataService.update(IMPORT_RUN, importRun);
+
+			}
+			catch (Exception e)
```
Comment (dennishendriksen) [inline molgenis-data-import/src/main/java/org/molgenis/data/importer/ImportRunService.java]: Catch RuntimeException or specific exceptions?

**Pull request merged**